sim/rogue/runes.go

```diff
@@ -153,3 +157,132 @@ func (rogue *Rogue) ApplyCutToTheChase(sim *core.Simulation) {
 		rogue.SliceAndDiceAura.Activate(sim)
 	}
 }
+
+// TODO: Update cut to the chase to refresh
```

Comment:
Looks like the duration refresh is still missing here
https://github.com/wowsims/sod/blob/79f61bb7d38978f4d6d81e1ae94e24cf3fa6f58d/sim/rogue/runes.go#L152-L159